### PR TITLE
feat: [GAPS-160]: Versioning - URL Manager - Configuration

### PR DESCRIPTION
**Pull request by Bernardvdv**
Added versioning configuration to cms_config

### CODE REVIEW

File: djangocms_url_manager/cms_config.py [hunk 1]
```diff
@@ -9,20 +9,50 @@
...
+    url_manager_supported_models_search_helpers = {
+        Page: get_page_search_results,
+    }
+    # djangocms-navigation settings
     djangocms_navigation_enabled = getattr(
         settings, "DJANGOCMS_NAVIGATION_CMS_MODELS_ENABLED", False
     )
     navigation_models = {Url: ["internal_name"]}
+    # djangocms-moderation settings
+    djangocms_moderation_enabled = getattr(
```
Comment (adam-murray) [inline djangocms_url_manager/cms_config.py:39]: The closing bracket is on a new line above, could we make these all consistent?

File: tests/test_admin.py [hunk 2]
```diff
@@ -39,9 +41,10 @@ def test_get_search_results_partial_search_term(self):
         self.assertEqual(results.last(), self.url2)
         self.assertEqual(results.count(), 2)
 
-    @skipUnless(
-        BaseUrlTestCase.is_versioning_enabled(), "Test only relevant for versioning"
-    )
+    # @skipUnless(
```
Comment (adam-murray) [inline tests/test_admin.py:44]: How come this is commented out?

**Review by adam-murray — APPROVED** [review]

File: djangocms_url_manager/cms_config.py (hunk 1 above)
Comment (Bernardvdv) [reply djangocms_url_manager/cms_config.py:39]: Done

File: tests/test_admin.py (hunk 2 above)
Comment (Bernardvdv) [reply tests/test_admin.py:44]: It should be uncommented once the reason for the test failing has been resolved. For now, it needs to be skipped

**Review by Aiky30 — APPROVED** [review]

**Comment by Aiky30** [comment]
Approved on the basis that all test skips will be fixed in the next change

**Pull request merged**